### ISSUE TITLE
Feature Request: Sites - display the website privacy status

### ISSUE THREAD

**Issue by Robertght** [Automattic/wp-calypso#96214]
### What

Coming from [here](https://github.com/Automattic/dotcom-feedback/issues/193):

>/sites shows the site launch/privacy status. This context is missing after you click into a site and land on its "Hosting Overview" page.

### Why

It should make it obvious for the user when they access it on which state it is at any time.

### How

I would add a label the site name and create a page jump link to the privacy setting to act as a a quick access to that page.

cc @lucasmendes-design

**Comment by inaikem** [comment]
@Copons, @taipeicoder 👋 With both your teams working on nav and settings, which team should I allocate this FR to? Thanks!

**Comment by taipeicoder** [comment]
I've assigned this to Lego, but ideally this should be assigned to the design team before assigning it to a dev team. Otherwise, it will stay on the dev team's backlog until it gets design input. Feels like a better workflow putting these discrete tasks in the design team backlog.

**Comment by inaikem** [comment]
Makes sense to me. @lucasmendes-design, Could we move this issue and others like it straight to the One Board's "Design" team?

The one missing piece of context PAs have here is which point we determine design is needed. For this issue, it was caught by Group Triage because it did not have a priority or team set.
